### ISSUE TITLE
[Popup Lyrics] Pause button is bugged in poup mode

### ISSUE THREAD

**Issue by Tetrax-10** [spicetify/spicetify-cli#2350]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: latest
- Spicetify version: latest commit
```

### 📝 Description

Pause button is bugged in poup mode

### 📸 Screenshots

![image](https://github.com/spicetify/spicetify-cli/assets/75513645/81894352-9f9c-49eb-b9ee-346babb56eb9)

**Comment by kyrie25** [comment]
Anything on the overlay is handled by the executable, we have no control over it.
Same as #1827, which was fixed in CEF 112 (Spotify 1.2.11)

**Comment by Tetrax-10** [comment]
ic
